alethzero-latest/tools/chocolateyInstall.ps1

```diff
@@ -1,20 +1,32 @@
-$packageName    = 'AlethZero-latest'

```

Comment:
Hello,

First of all, I really apologize for the delay. I did not have any time to check out anything about that topic yet.

The only thing that does not sound right is the change in the ```alethzero-stable``` file ```AlethZero-Win32-stable.7z```by ```Ethereum-win64-latest.exe```.

The rest looks very good and surely improves the existing.

If you could fix the issue mentioned above, I´d trust that you tested carefully and merge this PR that I should have merged long ago :smile: